### DISCUSSION TITLE
We need some report mechanism on the discussion

### DISCUSSION THREAD

**Discussion by flyleeee** [community/community#20657] [Discussions] [answered]
I think we need some report mechanism，because I have found some people put some pornographic nude pictures on the discussion😒

**Comment by radar-blip** [comment, marked as answer]
You can report them through their profile.

**Comment by depascalis** [comment]
Not only that but there are users who believes that the _Discussions_ feature is some kind of chat, creating threads that just say "Hello" for example. I have also started noticing accounts, fake or genuine?, posting useless answers like "Hello" or "Visit my profile" etc. Some users are just seeking attention or exposure, and  other users behave as they just found the internet (It's a mystery how they managed to create a GitHub account), either way, it's an improper or inappropriate use of the feature.

Discussions need some voting/reporting system in place that hides and/or deletes threads and answers that are either inappropriate or useless (like stackexchange.com for example). We are already seeing tendencies of pollution, so please act now before this feature is turned into a dump of trolling and spam.

**Comment by ashcash-maker** [comment]
I don’t know if it has something to do with my account but Amanda Valdez has been in multiple accounts of mine and other people as well stealing identity and cards and addresses everything .
